Makefile

```diff
@@ -321,6 +321,7 @@ purge: purge-doc purge-dev user-uninstall
 	rm -f "${DESTDIR}${BINDIR}/r2"
 	rm -f "${DESTDIR}${LIBDIR}/libr_"*
 	rm -f "${DESTDIR}${LIBDIR}/libr2.${EXT_SO}"
+	rm -f "${DESTDIR}${LIBDIR}/libr2."*".${EXT_SO}"
```

Comment:
just `rm -f "${DESTDIR}${LIBDIR}/libr2.*${EXT_SO}"` should be enough

Reply:
Well if you want to compress it into one line, I think that `rm -f "${DESTDIR}${LIBDIR}/libr2."*".${EXT_SO}"` would actually be a better alternative. Plus you cannot put a glob (such as the star) inside quotes.

Reply:
Do the change and ill merge

Reply:
Done.